sw/device/tests/aes_entropy_test.c

```diff
@@ -32,20 +28,24 @@ static const uint8_t kKey[] = {
     0x16, 0x17, 0x18, 0x19, 0x1a, 0x1b, 0x1c, 0x1d, 0x1e, 0x1f,
 };
 
-static const uint32_t kCipherTextGold[] = {
-    0x493350e6,
-    0x4a0a5568,
-    0x9d4d4aa1,
-    0xd256e2e0,
-};
-
 static const uint32_t kIv[] = {
     0x01020304,
     0x0c0d0e0f,
     0x1718191a,
     0x1c1d1e1f,
 };
 
+/**
+ * The following ciphertext are computed using the python library `PyCryptodome`
```

Comment:
If the code is short enough, can you add the python lines used to do the calculation in a comment here?

Reply:
Done!

Reply:
Great, thanks!